recipes/pytest-notebook/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pytest-notebook" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c6117a4ca0ec404ac31c28a4baa21b4a3031f0a6d635efee023d46b20de15be3"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py < 35]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - attrs
+    - black
+    - importlib_resources
+    - jsonschema
+    - jupyter_client
+    - nbconvert
+    - nbdime
+    - nbformat
+    - pytest >=3.5.0
+    - python
+
+test:
+  imports:
+    - pytest_notebook
+    - pytest_notebook.diffing
+    - pytest_notebook.example_nbs
+    - pytest_notebook.execution
+    - pytest_notebook.ipy_magic
+    - pytest_notebook.nb_regression
+    - pytest_notebook.plugin
+    - pytest_notebook.post_processors
+    - pytest_notebook.utils
+
+about:
+  home: "https://github.com/chrisjsewell/pytest-notebook"
+  license: "BSD"
+  license_family: "BSD"
+  license_file: "LICENSE"
+  summary: "A pytest plugin for testing Jupyter Notebooks"
+  doc_url: "https://pytest-notebook.readthedocs.io"
+  dev_url: ""
```

Comment:
Either add a dev_url or remove the entry as it is probably the same as home.

Reply:
Fixed